lib/open_project/translations/version.rb

```diff
@@ -14,6 +14,6 @@
 
 module OpenProject
   module Translations
-    VERSION = "5.0.0-alpha"
+    VERSION = '5.0.0-alpha'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.